src/reclaim_python_sdk/reclaim.py

```
@@ -321,50 +345,358 @@ def set_app_callback_url(self, url: str, json_proof_response: bool = False) -> N
 
         Raises:
-            AddContextError: If context cannot be added
+            AddContextError: When context cannot be added
+            InvalidParamError: When required parameters are missing
+
+        Example:
+            >>> proof_request.set_context('user-123', 'Account Verification Profile')
         """
         try:
             if not address or not message:
                 raise InvalidParamError("Address and message are required")
```

Ruff (0.15.5)

[warning] Abstract `raise` to an inner function

(TRY301)